android/src/main/java/com/reactnativestripesdk/StripeSdkCardView.kt

```diff
@@ -212,6 +212,7 @@ class StripeSdkCardView(private val context: ThemedReactContext) : FrameLayout(c
       override fun onCardComplete() {}
       override fun onExpirationComplete() {}
       override fun onCvcComplete() {}
+      override fun onPostalCodeComplete() {}
```

Comment:
adding a note for myself that we may want to add listeners for each of these events, not sure yet though